docs/buildah-push.md

```diff
@@ -52,6 +52,10 @@ The username[:password] to use to authenticate with the registry if required.
 
 Don't compress copies of filesystem layers which will be pushed.
 
+**--format, -f**
+
+`MANIFEST TYPE` (oci, v2s1, or v2s2) to use when saving image to directory using the 'dir:' transport (default is manifest type of source)
```

Comment:
May have missed this in earlier reviews, but I think this probably ought to be "Manifest type" or "Manifest Type" if you really want to make it stand out a little.

Reply:
fixed